### ISSUE TITLE
FreeBSD 12.0 broken?

### ISSUE THREAD

**Issue by Smattr** [cirruslabs/cirrus-ci-docs#632]
Since ~Apr 16 my FreeBSD 12.0 builds have been failing due to a pkg metadata mismatch:

```
pkg upgrade -y && pkg install -y bash bison cmake gmp libxml2 ninja z3
Updating FreeBSD repository catalogue...
Fetching meta.txz: . done
pkg: repository meta /var/db/pkg/FreeBSD.meta has wrong version 2
repository FreeBSD has no meta file, using default settings
Fetching packagesite.txz: .......... done
pkg: repository meta /var/db/pkg/FreeBSD.meta has wrong version 2
pkg: Repository FreeBSD load error: meta cannot be loaded No error: 0
Unable to open created repository FreeBSD
Unable to update repository FreeBSD
Error updating repositories!

Exit status: 3
```

This is using the 12.0 release:

```
# .cirrus.yml
...
  - name: FreeBSD 12.0
    freebsd_instance:
      image_family: freebsd-12-0
...
```

[FreeBSD 12.0 is out of support](https://www.freebsd.org/security/unsupported.html), so maybe this is expected? Is it just no longer possible to install new packages on the 12.0 images?

**Comment by fkorotkov** [comment]
Yeah, not sure what's going on there but there was another report and user fixed it by using 12.1. See #625.

**Comment by Smattr** [comment]
Thanks for the quick response, Fedor. I'm already testing on 12.1 so it's an easy fix to just drop 12.0.
